pkg/kritis/metadata/containeranalysis/containeranalysis_int_test.go

```diff
@@ -137,3 +140,30 @@ func TestCreateAttestationNoteAndOccurrence(t *testing.T) {
 		}
 	}
 }
+
+func TestGetMultiplePages_Vulnerabilities(t *testing.T) {
+	d, err := New()
+	if err != nil {
+		t.Fatalf("Could not initialize the client %s", err)
+	}
+
+	// Set PageSize to 300
+	createListOccurrencesRequest = createListOccurrencesRequest_test
+
+	vuln, err := d.Vulnerabilities("gcr.io/kritis-int-test/java-with-vulnz@sha256:358687cfd3ec8e1dfeb2bf51b5110e4e16f6df71f64fba01986f720b2fcba68a")
+	if err != nil {
+		t.Fatalf("Found err %s", err)
+	}
+
+	if len(vuln) <= 300 {
+		t.Errorf("Pagination error: only received results from first page.")
+	}
+}
+
+func createListOccurrencesRequest_test(containerImage, kind string) *grafeas.ListOccurrencesRequest {
```

Comment:
Let's keep the naming strategy consistent wrt camelCase vs snake_case

Reply:
Done.
Should I apply the same change to the top-level test `TestGetMultiplePages_Vulnerabilities`?